db/migrate/20990101999999_sample_migration.foreman_plugin_template.rb

```diff
@@ -0,0 +1,2 @@
+class SampleMigration < ActiveRecord::Migration
```

Comment:
Yes, docs are for examples. This ought to be an empty, usable plugin - not every plugin should add a new, empty migration.